hyperspy/utils/axes.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2020 The HyperSpy developers
+#
+# This file is part of HyperSpy.
+#
+# HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+import hyperspy.axes
+
+__doc__ = hyperspy.axes.__doc__
+
+def get_central_half_limits_of_axis(ax):
```

Comment:
This create a cyclic import somewhere:
```python
import hyperspy.roi
```

Reply:
Good catch. But I'm getting a headache just looking at it. I can easily solve it by just keeping it in `axes.py`, or by deleting `hyperspy/utils/roi.py` (it's got nothing in it?). But that feels like cheating.

Doesn't the existance of `import hyperspy.roi` in `hyperspy/utils/roi.py` mean that one cannot use it in `hyperspy/roi.py` without circular imports? That seems counterproductive?

Reply:
`hyperspy/utils/roi.py` defined what goes in the API through `hyperspy.api_nogui`, so I would leave it as it is for now!

Putting this function in `hyperspy.roi` should be good enough and possibly as a private function. Do you think it could used elsewhere?